pom.xml

```diff
@@ -43,18 +47,105 @@
     <module>hive-metastore-events</module>
   </modules>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>egdl-maven-snapshot-local</id>
+      <url>https://artylab.expedia.biz/egdl-maven-snapshot-local</url>
+    </snapshotRepository>
+    <repository>
+      <id>egdl-maven-release-local</id>
+      <url>https://artylab.expedia.biz/egdl-maven-release-local</url>
+    </repository>
+  </distributionManagement>
+
+
   <dependencyManagement>
```

Comment:
Pin all common dependencies here to treat the main parent pom as a bom instead of a pom